### ISSUE TITLE
Missing Measures after TabMon Installation

### ISSUE THREAD

**Issue by seanloftus** [tableau/TabMon#112]
After installation TabMon for our Tableau Production Environment, a number of Measures are missing from the TabMon Database.  Both the Server Auditing and TabMon Databases are up and running.  I can view data in both of them.  However under the TabMon Database, the following Measures are not found:

% Free Space
% Processor Time
%CPU Gradiant
Available Bytes
Bytes Received/Sec
Bytes Sent/Sec
Current Disk Queue Length
Disk Read Bytes/Sec
Disc Write Bytes/Sec
Pages Input/Sec
Private Bytes
Processor Queue Length

As you can imagine, this is causing problems with the Monitoring Dashboard.  Please let me know if you know what can be done to fix this.  Please see attached.

Thanks!
Sean
[TabMon Missing Measures.docx](https://github.com/tableau/TabMon/files/865701/TabMon.Missing.Measures.docx)

**Comment by seanloftus** [comment]
I found the issue.  After reading through the TabMon.log file, I found the following:

2017-03-24 15:07:58,771 [29] ERROR TabMon.Counters.Perfmon.PerfmonCounterLoader - Could not communicate with Perfmon on target host 'svrus4tabl1': Access is denied
2017-03-24 15:07:58,772 [29] DEBUG TabMon.Counters.Perfmon.PerfmonCounterLoader - PerfMon counter '% Processor Time' in category 'Process' on host 'svrus4tabl1' does not exist.
2017-03-24 15:07:58,777 [29] ERROR TabMon.Counters.Perfmon.PerfmonCounterLoader - Could not communicate with Perfmon on target host 'svrus4tabl1': Access is denied
2017-03-24 15:07:58,777 [29] DEBUG TabMon.Counters.Perfmon.PerfmonCounterLoader - PerfMon counter 'Private Bytes' in category 'Process' on host 'svrus4tabl1' does not exist.

This was being caused by permissions.  Once I added our service account to the "Performance Monitor Users" and "Performance Log Users" (like it says in the installation instructions), I was able to restart the TabMon  Services and everything is now coming through perfectly.  Thank you!
